pyproject.toml

```diff
@@ -14,10 +14,31 @@ docx-mailmerge = "^0.5.0"
 click = "^7.1.2"
 
 [tool.poetry.dev-dependencies]
```

Comment:
Note: for `nox` to work when installed as dev dependency you must invoke it via poetry, e.g. `poetry run nox`

Reply:
**Note:** `poetry run pre-commit install` in poetry's [contributing.md](https://github.com/python-poetry/poetry/blob/master/CONTRIBUTING.md), which is to ensure it's invoked within poetry's virtual environment rather than user level.

Reply:
Good point, I followed the guide which indicated it would be good to have these as global packages. For my understanding, I had to do `poetry install`, `poetry shell` and then `pre-commit install` in order for it to set up the pre-commit hooks (I could then exit the shell and it would work). Am I missing a shortcut?

Reply:
Woops - Now I get your comment about `poetry run pre-commit install`. Got it!